kinetic/distribution.yaml

```diff
@@ -1284,14 +1284,16 @@ repositories:
       - grid_map_filters
       - grid_map_loader
       - grid_map_msgs
+      - grid_map_pcl
       - grid_map_ros
       - grid_map_rviz_plugin
       - grid_map_visualization
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/ethz-asl/grid_map-release.git
-      version: 1.4.0-0
+      version: 1.4.1-1
     source:
+      test_pull_requests: true
```

Comment:
You granted push access to the `ethz-asl/grid_map` repository. I don't think that this will be sufficient. The docs say:

> Therefore the GitHub **org unit** where the repository is located must grant **push, pull and administrative rights** to the user `ros-pull-request-builder`.